.idea/codeStyleSettings.xml

```diff
@@ -0,0 +1,24 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
This is just project settings file, its different than IntelliJ global one, to populate something from IntelliJ to this file you have to declare it directly.